app/controllers/twiddles.js

```diff
@@ -0,0 +1,5 @@
+import Controller from '@ember/controller';
+
+export default class TwiddlesController extends Controller {
+  isSidebarOpen = true;
+}
```

Comment:
make sure your editor is following .editorconfig Specifically, insert_final_newline=true